cmd/config/config.go

```diff
@@ -1,82 +1,145 @@
 package main
 
 import (
-	"encoding/json"
 	"flag"
-	"log"
+	"fmt"
 	"os"
+	"slices"
+	"strings"
 
 	"github.com/thrasher-corp/gocryptotrader/common/file"
 	"github.com/thrasher-corp/gocryptotrader/config"
 )
 
-// EncryptOrDecrypt returns a string from a boolean
-func EncryptOrDecrypt(encrypt bool) string {
-	if encrypt {
-		return "encrypted"
-	}
-	return "decrypted"
-}
+var commands = []string{"upgrade", "encrypt", "decrypt"}
 
 func main() {
-	var inFile, outFile, key string
-	var encrypt bool
+	fmt.Println("GoCryptoTrader: config-helper tool")
+
 	defaultCfgFile := config.DefaultFilePath()
-	flag.StringVar(&inFile, "infile", defaultCfgFile, "The config input file to process.")
-	flag.StringVar(&outFile, "outfile", defaultCfgFile+".out", "The config output file.")
-	flag.BoolVar(&encrypt, "encrypt", true, "Whether to encrypt or decrypt.")
-	flag.StringVar(&key, "key", "", "The key to use for AES encryption.")
-	flag.Parse()
 
-	log.Println("GoCryptoTrader: config-helper tool.")
+	var in, out, key string
+	fs := flag.NewFlagSet("config", flag.ExitOnError)
+	fs.Usage = func() { usage(fs) }
+	fs.StringVar(&in, "infile", defaultCfgFile, "The config input file to process")
+	fs.StringVar(&out, "outfile", "[infile].out", "The config output file")
```

Comment:
I will forgive it  because the existing application was also unfriendly. No way to upgrade in place with a `config.vX.json` previous version output
Hate having to fiddle with files after doing the task.

I'm just ranting. This is something I can add later

Reply:
Yeah, I dislike this too.
I thought I'd get too much pushback if I changed it.
I'd like us to support unix centric as a starting point; Unless otherwise specified `-in` defaults to `-` meaning STDIN and `-out` defualts to `-` meaning STDOUT. If you specify `-in` but not `-out` you'd get output to STDOUT.
Add `-e | --edit` for edit-in-place.
I think that'd work for windows as well as unix.
I could do this now if it has support.

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/1b498f3bb

Mainly this gives you `-edit` to solve your problem.

This also fixes location of the command so it can be anywhere.
I didn't go so far as to switch to unixesque usage, because I don't think we have a use-case for input.